data/data/config.tf

```
@@ -123,3 +123,14 @@ Should the cluster be created with ipv6 networking.
 EOF
 
 }
+
+variable "default_description" {
+  type = string
+
+  description = <<EOF
+Default description that should show for the components
+EOF
+
+default = "Created by Openshift Installer"
```

This needs leading spaces. And the s in OpenShift should be capitalized.
```suggestion
  default = "Created by OpenShift Installer"
```